content/en/blog/_posts/2025-01-14-devices-failure-handling/index.md

```diff
@@ -0,0 +1,409 @@
+---
+layout: blog
+title: "Navigating Failures in Pods With Devices"
+date: 2025-01-14
+slug: navigating-failures-in-pods-with-devices
+author: >
+  Sergey Kanzhelev (Google)
+  Mrunal Patel (RedHat)
```

Comment:
Hi. We're recommending that we merge articles as draft (`draft: true`) and then you send in a short follow-up PR that sets the final publication date and marks it as not-draft.

Please add `draft: true` here.

Reply:
done